examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -57,6 +57,10 @@
 rng = np.random.RandomState(0)
 
 X_full, y_full = fetch_california_housing(return_X_y=True)
+# ~2k samples is enough for the purpose of the example.
+# Remove the following two lines for a slower run with different error bars
```

Comment:
``different`` -> ``similar``? And a ``.`` at the end.

Reply:
The error bars are different, not really similar.

Reply:
Really, could you please post these two figure?

Reply:
before
![before](https://user-images.githubusercontent.com/1663864/53746721-1cc7f000-3ea2-11e9-8113-0b0acd33e795.png)

after
![after](https://user-images.githubusercontent.com/1663864/53746718-1cc7f000-3ea2-11e9-8a52-3d727159ab0a.png)

Reply:
So you think they are not similar?

Reply:
exactly. Specially for the last one and bayesridge